code/modules/mob/living/living.dm

```diff
@@ -792,3 +793,10 @@ default behaviour is:
 		update_icons()
 	return canmove
 
+/mob/living/proc/update_water() // Involves overlays for humans.  Maybe we'll get submerged sprites for borgs in the future?
+	return
```

Comment:
It's there so other things can inherit it, like borgs, and how borg icon code is different from human icon code unfortunately.